src/autoscaler/metricscollector/db/metricscollector.db.changelog.yml

```diff
@@ -30,4 +30,46 @@ databaseChangeLog:
                   name: value
                   type: varchar
                   constraints:
-                    nullable: true
\ No newline at end of file
+                    nullable: false
+  - changeSet:
+      id: 2
+      author: byang
+      changes:
+        - createTable:
+            tableName: appinstancemetrics
```

Comment:
I believe we are missing indexes based on how we query and delete.
You should do an explain and see how much a timestamp helps. Name would be another.